shared/aggregation/attestations/maxcover.go

```diff
@@ -110,17 +93,7 @@ func (al attList) aggregate(coverage bitfield.Bitlist) (*ethpb.Attestation, erro
 
 // merge combines two attestation lists into one.
 func (al attList) merge(al1 attList) attList {
-	merged := make([]*ethpb.Attestation, len(al)+len(al1))
-	i := 0
-	for _, att := range al {
-		merged[i] = att
-		i++
-	}
-	for _, att := range al1 {
-		merged[i] = att
-		i++
-	}
-	return merged
+	return append(al, al1...)
```

Comment:
I am not sure what I was thinking about when I implemented manual merging of two slices, when `append()` has us covered (if future the whole `attList.merge()` method might be removed, and we will just be inlining `append()`, but for now wanted to keep change constrained).